labelbox/data/serialization/coco/panoptic_dataset.py

```diff
@@ -0,0 +1,178 @@
+from labelbox.data.annotation_types.geometry import Polygon, Rectangle
+from labelbox.data.annotation_types import Label
+from labelbox.data.annotation_types.geometry.mask import Mask
+from labelbox.data.annotation_types.annotation import ObjectAnnotation
+from labelbox.data.annotation_types.data.raster import MaskData, ImageData
+from labelbox.data.serialization.coco.categories import Categories, hash_category_name
+from labelbox.data.annotation_types.collection import LabelCollection
+from labelbox.data.serialization.coco.image import CocoImage, get_image, get_image_id, id_to_rgb, rgb_to_id
+from labelbox.data.serialization.coco.annotation import PanopticAnnotation, SegmentInfo, get_annotation_lookup
+from concurrent.futures import ProcessPoolExecutor, as_completed
+from pydantic import BaseModel
+from tqdm import tqdm
+import os
+import numpy as np
+from PIL import Image
+from typing import Dict, Any, List, Union
+
+
+def vector_to_coco_segment_info(canvas: np.ndarray,
+                                annotation: ObjectAnnotation,
+                                annotation_idx: int, image: CocoImage,
+                                category_id: int):
+    shapely = annotation.value.shapely
+    xmin, ymin, xmax, ymax = shapely.bounds
+    canvas = annotation.value.draw(height=image.height,
```

Comment:
`value` seems to be of type `Geometry` - I don't see it having `draw` method?

Reply:
Yeah it has a draw method:
https://github.com/Labelbox/labelbox-python/blob/7432b6615e35130a30c8ee8f549c96a97e4686b0/labelbox/data/annotation_types/geometry/geometry.py#L38